src/Video.tsx

```diff
@@ -315,6 +318,24 @@ export const RemotionVideo: React.FC = () => {
 								'https://s3.eu-west-3.amazonaws.com/moovijob.prod/1494438/Horizontal_Black_Logo-Zenika.png',
 						}}
 					/>
+					<Still
```

Comment:
We definitely need to split this file in more than one and use lazyComponent import for loading time improvement.

I will definitely work on that in #155

Reply:
Yes definitely ! We have to check that !